src/scenes/ObservationsRouter/org/OrgObservationsListView.tsx

```diff
@@ -123,6 +69,66 @@ export default function OrgObservationsListView({
   const [requestId, setRequestId] = useState('');
   const abandonObservationResponse = useAppSelector((state) => selectAbandonObservation(state, requestId));
   const snackbar = useSnackbar();
+  const isSurvivalRateCalculationEnabled = isEnabled('Survival Rate Calculation');
+
+  const defaultColumns = useCallback(
+    () =>
```

Comment:
Could this one change to a `useMemo`, or does the table need a callback?

Reply:
Yes, this can be a useMemo